scss/detail.scss

```diff
@@ -0,0 +1,40 @@
+$gray : #828282;
+
+.book {
+  display: flex;
+  font-family: 'Roboto', sans-serif;
+  flex-direction: column;
+  padding: 15px;
+}
+
+.row {
+  padding: 18px 0;
+  display: flex;
+  flex-direction: row;
```

Comment:
It's awesome that you already thought of making a helper class for this. Although, I would leave padding out of this to make it more reusable, also there is no need to add `flex-direction: row`, when you set `display: flex` this sets the direction as row as default. 

Also, you should define a class for column and reuse that too